aws/resource_aws_gamelift_build_test.go

```diff
@@ -0,0 +1,302 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/gamelift"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_gamelift_build", &resource.Sweeper{
+		Name: "aws_gamelift_build",
+		F:    testSweepGameliftBuilds,
+	})
+}
+
+func testSweepGameliftBuilds(region string) error {
+	client, err := sharedClientForRegion(region)
+	if err != nil {
+		return fmt.Errorf("error getting client: %s", err)
+	}
+	conn := client.(*AWSClient).gameliftconn
+
+	resp, err := conn.ListBuilds(&gamelift.ListBuildsInput{})
+	if err != nil {
+		return fmt.Errorf("Error listing Gamelift Builds: %s", err)
+	}
+
+	if len(resp.Builds) == 0 {
+		log.Print("[DEBUG] No Gamelift Builds to sweep")
+		return nil
+	}
+
+	log.Printf("[INFO] Found %d Gamelift Builds", len(resp.Builds))
+
+	for _, build := range resp.Builds {
+		if !strings.HasPrefix(*build.Name, "tf_acc_build_") {
```

Comment:
Nitpick: I think we should use a constant here if we're going to expect continuity between the acceptance test naming and the sweeper.

Reply:
Yeah, that's a good idea.